easybuild/easyconfigs/s/SAMtools/SAMtools-1.6-foss-2017a.eb

```diff
@@ -0,0 +1,38 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+#
+# Copyright:: Copyright 2012-2014 Uni.Lu/LCSB, NTUA
+# Authors::   Robert Schmidt <roschmidt@ohri.ca>, Cedric Laczny <cedric.laczny@uni.lu>, Fotis Georgatos <fotis@cern.ch>
+# License::   MIT/GPL
+# $Id$
+#
+# Modified by: Adam Huffman
+# The Francis Crick Institute
+#
+# Modified for version 1.4 by: Kurt Lust, UAntwerpen
+#
```

Comment:
oops, forgot to add to this. or should ew remove this header completely?

Reply:
better leave the header in-place, to retain attribution

whether you add yourself or not, is up to you, we don't really have a policy on this...